react/index.jsx

```diff
@@ -197,7 +197,7 @@ export default class SocialLogo extends PureComponent {
 				svg = <svg className={ iconClass } height={ size } width={ size } onClick={ onClick } { ...otherProps } xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M12.158 12.786L9.46 20.625a8.984 8.984 0 005.526-.144.852.852 0 01-.065-.124l-2.763-7.571zM3.009 12a8.993 8.993 0 005.067 8.092L3.788 8.341A8.952 8.952 0 003.009 12zm15.06-.454c0-1.112-.399-1.881-.741-2.48-.456-.741-.883-1.368-.883-2.109 0-.826.627-1.596 1.51-1.596.04 0 .078.005.116.007A8.963 8.963 0 0012 3.009a8.982 8.982 0 00-7.512 4.052c.211.007.41.011.579.011.94 0 2.396-.114 2.396-.114.484-.028.541.684.057.741 0 0-.487.057-1.029.085l3.274 9.739 1.968-5.901-1.401-3.838c-.484-.028-.943-.085-.943-.085-.485-.029-.428-.769.057-.741 0 0 1.484.114 2.368.114.94 0 2.397-.114 2.397-.114.485-.028.542.684.057.741 0 0-.488.057-1.029.085l3.249 9.665.897-2.996c.456-1.169.684-2.137.684-2.907zm1.82-3.86c.039.286.06.593.06.924 0 .912-.171 1.938-.684 3.22l-2.746 7.94a8.984 8.984 0 004.47-7.771 8.922 8.922 0 00-1.1-4.313zM12 22C6.486 22 2 17.514 2 12S6.486 2 12 2s10 4.486 10 10-4.486 10-10 10z"/></g></svg>;
 				break;
 			case 'x':
-				svg = <svg className={ iconClass } height={ size } width={ size } onClick={ onClick } { ...otherProps } ariaLabel="Threads" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g><path d="M14.117 9.622L20.446 2h-1.5l-5.495 6.618L9.062 2H4l6.637 10.007L4 20h1.5l5.803-6.989L15.938 20H21L14.117 9.622zm-2.054 2.474l-.672-.997-5.35-7.93h2.303l4.318 6.4.672.996 5.613 8.319h-2.304l-4.58-6.788z"/></g></svg>;
+				svg = <svg className={ iconClass } height={ size } width={ size } onClick={ onClick } { ...otherProps } ariaLabel="X" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g><path d="M14.117 9.622L20.446 2h-1.5l-5.495 6.618L9.062 2H4l6.637 10.007L4 20h1.5l5.803-6.989L15.938 20H21L14.117 9.622zm-2.054 2.474l-.672-.997-5.35-7.93h2.303l4.318 6.4.672.996 5.613 8.319h-2.304l-4.58-6.788z"/></g></svg>;
```

Comment:
Also, this should be `aria-label` instead of `ariaLabel`. It's the same for `threads` icon.

It results in JS warning

```
Warning: Invalid ARIA attribute `ariaLabel`. Did you mean `aria-label`?
```

<img width="653" alt="Screenshot 2023-10-12 at 12 40 24 PM" src="https://github.com/Automattic/social-logos/assets/18226415/22761c61-9920-477a-a11e-e1d1703d32f2">

Reply:
Thanks for letting me know!

I just pushed a new release of the package including #128, and I'll update Jetpack in https://github.com/Automattic/jetpack/pull/33591